cypress/support/e2e.js

```diff
@@ -48,3 +48,14 @@ import './commands/toolbar.js'
 
 // Assertions
 import './assertions/expect_title.js'
+
+// This is needed to prevent Cypress tests from failing due to the error:
+// Uncaught TypeError: Cannot read properties of undefined (reading 'received')
+//     at push../node_modules/actioncable/lib/assets/compiled/action_cable.js.ActionCable.Subscriptions.Subscriptions.notify (vendor-5d5d9b46ecb6acf021a4.js:280600:45)
+//     at Connection.message (vendor-5d5d9b46ecb6acf021a4.js:280454:43)
+// This error occurs on the dashboard usually the first time ManageIQ loads and is causing the Cypress tests to fail so we need this function to prevent that behaviour
+Cypress.on('uncaught:exception', (err, runnable) => {
+    // returning false here prevents Cypress from
+    // failing the test
+    return false
```

Comment:
I wonder if we should check the error message and be selective - this will allow all uncaught exceptions through, which probably shouldn't happen

Even better, if we can prevent the error from happening that would be better.

Reply:
We decided on making the error handling more selective until this can eventually be fixed. Believe it to be an action cable error on the back end.